pkg/apis/db/v1alpha1/cassandracluster_types.go

```diff
@@ -754,7 +769,7 @@ type CassandraClusterSpec struct {
 	// Pod defines the policy for pods owned by cassandra operator.
 	// This field cannot be updated once the CR is created.
 	//Pod       *PodPolicy         `json:"pod,omitempty"`
-	Resources CassandraResources `json:"resources,omitempty"`
+	Resources *v1.ResourceRequirements `json:"resources,omitempty"`
```

Comment:
Actually I was asking myself the same question..
It might be useful for these kinds of set in yaml definitions ? (but these existed with `CassandraRessources` without pointer so it must be wrong but just in case)

https://github.com/Orange-OpenSource/casskop/blob/4c0e2cc25068c4d8753ec734640c238c15926496/test/e2e/testdata/cassandracluster-2DC.yaml#L20

I've actually taken the definition you dropped on Slack, with '*', and kept it. If it's not justified by any needs I can remove it (and refactor usages).

Reply:
I've made the switch to non-pointer type. Single commit so easy revert possible.

Reply:
I've made the changes but using non pointer type makes all YAML to have a `resources:{}` in all DCs. And I had to replace https://github.com/Orange-OpenSource/casskop/pull/279/files#diff-e38c00543f76c06f16d7f16e0e473252f84738a0c6315ceb4839692a5bf255c4R200 with `if dcResources.Limits.Cpu().IsZero() && dcResources.Limits.Memory().IsZero() && dcResources.Requests.Cpu().IsZero() && dcResources.Requests.Memory().IsZero(){
` because `=`.

If this is incorrect, I can revert commit to go with pointer version.